WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -120,11 +152,9 @@ class MySiteViewModel @Inject constructor(
         _onNavigation.value = Event(OpenSitePicker(site))
     }
 
-    fun updateSite(selectedSite: SiteModel?) {
-        _selectedSite.value = selectedSite
-    }
-
-    fun refreshAccountAvatarUrl() {
+    fun refresh(selectedSite: SiteModel?) {
+        selectedSiteRepository.updateSite(selectedSite)
```

Comment:
To illustrate what I mentioned in the other comment, wouldn't calling `selectedSiteRepository.updateSite(selectedSiteRepository.getSelectedSite())` here essentially produce the same result?

I'm assuming we're doing this to force a refresh in the repository live data, in which case we could even consider introducing a `SelectedSiteRepository::forceRefresh`.

Reply:
I don't think this call is necessary at the moment. It's a leftover from when we weren't using the repository. I'll remove it. Updating site settings should be enough 👍